packages/app/src/systems/Asset/components/AssetItem/AssetItemAmount.tsx

```diff
@@ -37,16 +39,30 @@ export const AssetItemAmount = ({
   return (
     <Tooltip content={original.display} delayDuration={0} open={open}>
       <Box css={styles.root}>
-        <Text as="span" ref={amountRef} css={styles.amount}>
-          <AmountVisibility
-            value={amount}
-            units={decimals}
-            visibility={visibility}
-          />
-        </Text>
-        <Text as="span" css={styles.symbol}>
-          {symbol}
-        </Text>
+        <VStack gap="0">
+          <Text
+            as="span"
+            ref={amountRef}
+            css={styles.amount}
+            aria-label={`${symbol} token balance`}
+          >
+            <AmountVisibility
+              value={amount}
+              units={decimals}
+              visibility={visibility}
+            />
+            <Text as="span" css={styles.symbol}>
+              {symbol}
+            </Text>
+          </Text>
+          <Text
+            aria-hidden={visibility}
+            aria-label={`${symbol} conversion rate to USD`}
+            css={styles.amountInUsd}
+          >
+            {visibility ? (amountInUsd ?? '$0.00') : '•••••'}
```

Comment:
like before we had "ETH ...." can we keep the $ when it's not visible also? "$ ...."

![image](https://github.com/user-attachments/assets/22de561c-50c6-4ab6-92d4-8dd46730b92f)

Reply:
Ofc!